workflow/workflow_commands.go

```diff
@@ -1445,6 +1448,76 @@ func TraceWorkflow(c *cli.Context) error {
 	return nil
 }
 
+func UpdateWorkflow(c *cli.Context) error {
+	wid := c.String(common.FlagWorkflowID)
+	rid := c.String(common.FlagRunID)
+	name := c.String(common.FlagName)
+	uid := c.String(common.FlagUpdateID)
+	first_execution_run_id := c.String(common.FlagUpdateFirstExecutionRunID)
```

Comment:
ditto on camel case

Reply:
Are you talking about CLI and ID? Those would fall under the [initialism exception](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).